docs/reference/bare_metal/microlib.md

```diff
@@ -0,0 +1,52 @@
+# Using ARM microlib in Mbed OS
+
+## What is microlib
+Microlib is an alternative library to the default C library. It is intended for use with deeply embedded applications that must fit into extremely small memory footprints.
+These applications do not run under an operating system. More information can be found [here](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0808e/chr1358938937854.html).
+
+## Differences between ARM C standard library and microlib
+There are a number of differences between microlib and the default C library. Follow [this link](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0808e/chr1358938938431.html) for a complete list.
```

Comment:
Can you replace the link with https://developer.arm.com/docs/100073/0613/the-arm-c-micro-library/differences-between-microlib-and-the-default-c-library as what you linked is the old documentation website.